### DISCUSSION TITLE
session-start: ** ERROR ** Failed to start new session: Failed calling D-Bus method

### DISCUSSION THREAD

**Discussion by JahanzebNawaz** [OpenVPN/openvpn3-linux#199] [Q&A - Our support arena]
I just updated my ubuntu 22.04 LTS jammy  to ubuntu lunar 23.04  and now I'm having issues with starting my openvpn3. there were several issue first, I reinstalled it  but now I'm having session-start issue. not working have explored the net but none of the solutions worked till now.

Ubuntu:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 23.04
Release:	23.04
Codename:	lunar
```

command and error

```openvpn3 session-start --config config.ovpn```

error

```Using configuration profile from file:  config.ovpn
Session path: /net/openvpn/v3/sessions/3cdd72b7s7df9s4ba3s9e57sa6a186276afc
Private key passphrase: 
session-start: ** ERROR ** Failed to start new session: Failed calling D-Bus method Ready: GDBus.Error:net.openvpn.v3.sessions.error: Backend VPN process has died.  Session is no longer valid.
```

**Comment by dsommers** [comment]
This is a duplicate of #177.  It's a known issue, and we will try to solve that soon.
